napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -112,13 +111,13 @@ def changeBlending(self, text):
             self.layer.blending not in NO_OPACITY_BLENDING_MODES
         )
 
-        self.blendComboBox.setToolTip(
-            MIN_BLENDING_TOOLTIP if self.layer.blending == 'minimum' else ''
-        )
+        blending_tooltip = ''
         if self.layer.blending == 'minimum':
```

Comment:
```suggestion
        if self.layer.blending == str(Blending.MINIMUM):
```
I know that this is out of the context of this PR but increases code consistency.